src/Imath/ImathVec.h

```diff
@@ -72,15 +72,24 @@ template <class T> class IMATH_EXPORT_TEMPLATE_TYPE Vec2
     IMATH_HOSTDEVICE constexpr Vec2 (T a, T b) IMATH_NOEXCEPT;
 
     /// Copy constructor
+#if IMATH_VEC_BE_TRIVIALLY_RELOCATABLE != 0
+    IMATH_HOSTDEVICE constexpr Vec2 (const Vec2& v) IMATH_NOEXCEPT = default;
+#else
     IMATH_HOSTDEVICE constexpr Vec2 (const Vec2& v) IMATH_NOEXCEPT;
+#endif
 
     /// Construct from Vec2 of another base type
     template <class S>
     IMATH_HOSTDEVICE constexpr Vec2 (const Vec2<S>& v) IMATH_NOEXCEPT;
 
-    /// Assignment
+    /// Copy assignment
+#if IMATH_VEC_BE_TRIVIALLY_RELOCATABLE != 0
+    IMATH_HOSTDEVICE IMATH_CONSTEXPR14 Vec2&
+    operator= (const Vec2& v) IMATH_NOEXCEPT = default;
+#else
     IMATH_HOSTDEVICE IMATH_CONSTEXPR14 const Vec2&
     operator= (const Vec2& v) IMATH_NOEXCEPT;
+#endif
 
     /// Destructor
     ~Vec2 () IMATH_NOEXCEPT = default;
```

Comment:
```suggestion
    ~Vec2 () = default;
```
Destructors are noexcept by default; you don't have to write it. (Unless `T` has a throwing destructor, in which case your `noexcept` here _was_ actually doing something... but in that case, I'd say you have bigger problems.)